test/concatAll.js

```diff
@@ -0,0 +1,30 @@
+import { assert } from 'chai';
+
+import { concatAll } from '../src';
+import eq from './shared/eq';
+
+describe('concatAll', function() {
+  it('concatenates arrays', function() {
```

Comment:
Prefer should notations;  the same applies for other tests
```js
it('should concatenate arrays'

Reply:
Done.